.remarkrc

```diff
@@ -0,0 +1,12 @@
+{
+  "settings": {
+    "bullet": "-",
+    "fences": true,
+    "listItemIndent": "1"
+  },
+  "plugins": [
+    "remark-frontmatter",
+    "remark-preset-lint-recommended",
+    ["lint-list-item-indent", "space"] 
+  ]
+}
```

Comment:
est-ce qu'on ne pourrait pas mutualiser cette config, ou au moins les rules ? Je crois qu'on a fait également cet ajout de remark sur un autre dépôt ?

Reply:
Done in https://github.com/cozy/cozy-libs/pull/1112